cabal-testsuite/cabal-testsuite.cabal

```diff
@@ -28,7 +28,7 @@ common shared
   build-depends:
     , base >= 4.6 && <4.16
     -- this needs to match the in-tree lib:Cabal version
-    , Cabal == 3.5.0.0
+    , Cabal ^>= 3.6.0.0
```

Comment:
> this needs to match the in-tree lib:Cabal version

i think the operator should remain `==` then

Reply:
Done, thanks. I read that as "should match in practice"